bin/swagger-project.js

```diff
@@ -63,6 +65,14 @@ app
   .option('-b, --debug-brk [port]', 'start in remote debug mode, wait for debugger connect')
   .option('-m, --mock', 'run in mock mode')
   .action(execute(project.test));
+  
+app 
+  .command('generate-test [directory]')
+  .description('Generate the test template')
+  .option('-p, --path-name [path]', 'a sepecific path of the api, also suppport regular expression')
+  .option('-f, --test-module <module>', 'one of: ' + testmodules, 'supertest')
```

Comment:
Don't hardcode `"supertest"` and other defaults 
` .option('-f, --test-module <module>', 'one of: ' + testmodules, 'supertest')` => ` .option('-f, --test-module <module>', 'one of: ' + testmodules,  project.testmodules[0])`